etna/search/forms.py

```diff
@@ -189,7 +191,19 @@ class BaseCollectionSearchForm(forms.Form):
         default_month=1,
     )
     opening_end_date = DateInputField(
-        label="To",
+        label="to",
+        required=False,
+        default_day=END_OF_MONTH,
+        default_month=12,
+    )
+    created_start_date = DateInputField(
+        label="From",
```

Comment:
Any reason why we're mixing case for labels here? In my experience, form field labels usually start with an upper-case letter.

Reply:
Why even ask them? You certainly know how to make like difficult for yourself :)